scripts/patch-fn-serviceaccount.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+kubectl patch serviceaccount default -p '{"imagePullSecrets": [{"name": "registry-secret"}]}' -n openfaas-fn
```

Comment:
I would move this command in go, same as you did with the secrets commands.

Reply:
Thank you for taking a look. Other than running this through Go rather than a shell script, the approach seems sound?

Reply:
We assume the local docker config has been configure with the OF registry but this could not be the case. The alternative would be to ask for a url/user/pass with a command prompt.

Reply:
That's true. The instructions in the README and dev-guide for OpenFaaS Cloud mandate that you've run `docker login`. It's one of the easiest ways to pre-populate the data.